FlowCrypt/Functionality/Services/EnterpriseServerApi.swift

```diff
@@ -22,6 +25,14 @@ class EnterpriseServerApi: EnterpriseServerApiType {
         static let serviceNeededValue = "enterprise-server"
     }
 
+    private struct ClientConfiguration: Codable {
+        let domainRules: DomainRules
```

Comment:
let's be consistent in naming - if the incoming attribute is called `clientConfiguration` then we should call it the same. That means `clientConfiguration: ClientConfiguration`. You can rename this whole struct to `ClientConfigurationApiResponse` to differentiate.